docs/tutorials/notebooks/Monge_Gap.ipynb

```diff
@@ -0,0 +1,734 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #35.    dict_monge_gaps = {
</pre><p>I'd slightly prefer to defined 2 variables for the 2 different Monge gaps and later use:</p><p>for key, monge_gap in zip(["no_monge_gap", ...], [None, ]):</p><p>    ...</p><p>But if you prefer this, let's keep it!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>